brotli.go

```diff
@@ -0,0 +1,22 @@
+package httpgzip
+
+import (
+	"io"
+	"net/http"
+)
+
+func (fs *fileServer) findBrotliFile(w http.ResponseWriter, r *http.Request, fpath string) io.ReadSeeker {
+	brotliFilePath := fpath + ".br"
+
+	if file, err := fs.root.Open(brotliFilePath); err == nil {
+		defer file.Close()
```

Comment:
I think you would not need to close the file, given the HTTP stream should do that 🤔  @unknwon maybe you know better? Not super familiar with this stuff.

Yeah `Open` is the only method on an http.FS. I don't expect it to be slower than a stat call though, because in the err case it should be just as fast, and otherwise we actually need to open the file anyways (and not _just_ stat). This is the implementation of `Open` we're using and it doesn't seem to be overly complex and reaches out to `os.Open` right away: https://cs.opensource.google/go/go/+/refs/tags/go1.17.2:src/net/http/fs.go;l=71-84